pinot-broker/src/main/java/org/apache/pinot/broker/api/resources/PinotClientRequest.java

```diff
@@ -63,11 +63,18 @@
   @Inject
   private BrokerMetrics brokerMetrics;
 
+  /**
+   * Legacy API to query Pinot using PQL (Pinot Query Language) syntax
+   * and semantics. This API is deprecated and PQL is no longer supported
+   * by Pinot. The API will be removed in the next release. Please use
+   * the standard SQL syntax (API /query/sql) to query Pinot.
+   */
```

Comment:
Is there any public doc on how to migrate PQL to SQL? It'd be good to link it here for users to migrate if needed.

Reply:
Good point. I had created an internal wiki for Li users to migrate from PQL to SQL. May be I can extract some parts of it and create an open source documentation in docs.pinot.apache.org.